skeleton/styles/prosilver/template/mcp_demo_body.html.tpl

```diff
@@ -0,0 +1,27 @@
+&lt;!-- INCLUDE mcp_header.html -->
+
+<form method="post" id="mcp" action="&#123;U_POST_ACTION}">
```

Comment:
It has to do this, to prevent the skeleton packager from accidentally parsing this as template syntax.

See: https://github.com/phpbb-extensions/phpbb-ext-skeleton/blob/3.1.x/helper/packager.php#L183-L184